docs/articles/user-guide/development-models.md

```diff
@@ -0,0 +1,209 @@
+---
+title: Development Models with VS Code
+---
+
+## Overview
+
+Salesforce Extensions for VS Code supports Org Development and Package Development models to authorize, create and switch orgs in your project:
+
+- **Org Development Model**: Allows you work with orgs that don’t have source tracking, such as sandbox, Developer Edition (DE) org, Trailhead Playground, or even a production org to retrieve and deploy code directly.
+- **Package Development Model**: Allows you to create self-contained applications or libraries that are deployed to your org as a single package. These packages are typically developed against source-tracked orgs such as scratch orgs. This development model uses org source tracking, source control, and continuous integration and deployment.
+
+Salesforce Extensions for VS Code runs commands against the org that you’ve set as your default org for development.
+
+## Org Development Model
```

Comment:
This article is really long. Do you not think it is too much for one document?

Reply:
The heading in the right pane makes it to easy to navigate. Also, deleted the CLI commands section from this topic.